### DISCUSSION TITLE
errors running ne120 grid

### DISCUSSION THREAD

**Discussion by anasena** [E3SM-Project/E3SM#4634] [E3SM model help]
Hi!
I am trying to run the F20TR-CICE compset with a forcing file with monthly SST/SICE at 0.25o horizontal resolution and I am running into some errors.
I did a few test runs with different grids. In all cases, the model builds and starts running, but crashes after 20-50 minutes. I ran into 3 different types of errors:
 
1) In ne120_ne120, ne120_g16 and ne120_oRRS15 I get the same error: “mismatch of input dimensions 933206 with expected value 233544 for variable topounit.” 
The land log points to the file /compyfs/inputdata/lnd/clm2/initdata_map/clmi.I1850CLM45.ne120_g16.c7cfd71.clm2.r.0021-01-01-00000.nc for the ne120_ne120 and ne120_g16 grids and to the file /compyfs/inputdata/lnd/clm2/initdata_map/clmi.I1850CLM45.ne120_oRRS15.c7cfd71.clm2.r.0021-01-01-00000.nc for the ne120_oRRS15 grid. 

2) In both ne120pg2_r0125_oRRS15to6v3 and ne120_r0125_oRRS18to6v3, I get a SIGTERM error. I don’t have a more specific error in this case, but I am attaching the log file for the latter grid. 

3) Finally, in ne120pg2_r05_oECv3 I get: (shr_stream_findBonds) ERROR: LVD not found, all data is before yearFirst.

Am I missing something about those grids/compset/machine?

[ne120_r0125_oRRS18to6v3-e3sm.log.242319.211101-142220.txt](https://github.com/E3SM-Project/E3SM/files/7463355/ne120_r0125_oRRS18to6v3-e3sm.log.242319.211101-142220.txt)

**Comment by jonbob** [comment]
@anasena - it might help if you point us at the case and run directories. The errors in (2) may simply be memory issues -- those are both highly-refined tests. And (3) may be issues in your forcing stream, though it's impossible to know without more information

**Reply by anasena** [reply]
@jonbob Sure! Thank you!
The forcing file in these tests is /compyfs/thom166/Scripts/Forcings/2010/SST-forcing-clim-pres-HR.nc
There is a similar forcing that worked in a different version/compset in /compyfs/thom166/Scripts/Forcings/2000/SST-forcing-clim-pres.nc

The case and run directories are:
1)	ne120_g16:  
casedir: /compyfs/thom166/cases/F20TR-CICE-ne120_g16-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/F20TR-CICE-ne120_g16-E3SMv2-1/run/

       ne120_ne120:  
casedir: /compyfs/thom166/cases/F20TR-CICE-ne120_ne120-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/F20TR-CICE-ne120_ne120-E3SMv2-1/run/

       ne120_oRRS15:  
casedir: /compyfs/thom166/cases/F20TR-CICE-ne120_oRRS15-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/F20TR-CICE-ne120_ oRRS15-E3SMv2-1/run/

2)	ne120pg2_r0125_oRRS18to6v3:
casedir: /compyfs/thom166/cases/F20TR-CICE- ne120pg2_r0125_oRRS18to6v3-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/F20TR-CICE- ne120pg2_r0125_oRRS18to6v3-E3SMv2-1/run/

ne120_r0125_oRRS18to6v3:
casedir: /compyfs/thom166/cases/F20TR-CICE ne120_r0125_oRRS18to6v3-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/F20TR-CICE ne120_r0125_oRRS18to6v3-E3SMv2-1/run/

3)	ne120pg2_r05_oECv3:
casedir: /compyfs/thom166/cases/F20TR-CICE-ne120pg2_r05_oECv3-E3SMv2-1/
rundir: /compyfs/thom166/e3sm_scratch/ F20TR-CICE-ne120pg2_r05_oECv3-E3SMv2-1/run/

Thanks!

**Reply by jonbob** [reply]
Thanks -- that will help. I looked at (3) quickly and the error message you're seeing is from docn and indicates a problem with the stream you set up. I looked at the dates in your forcing file and it shows:
````
 date = 116, 215, 316, 416, 516, 616, 716, 816, 916, 1016, 1116, 1216 ;
````
Is there a reason you're trying to run from a startup at 20000401 with this data? It is set to cycle, but the error you're getting shows the code complaining about the stream dates being below the lowest bound. I think the stream you have is inconsistent with the compset, which assumes a stream that encompasses the dates between 1869 and 2016 -- with what you have it can't cycle to get to the current date. You will need to use a compset that does not have EAM in its CMIP6 mode, I think

**Comment by mt5555** [comment]
A note about these CMIP6 compsets: these are complex time-dependent forcings that E3SM has setup for only a single resolution for V1, and a slightly different resolution for V2.  You are using higher resolution, where it is most likely these compsets wont work, nor are they supported.  Some of the other simpler compsets may work better.   For the V2 release in particular, we didn't release any ne120 configurations, only ne30 and RRM.   For V1, I think there was a ne120 configuration, but only for high-res MIP, not CMIP6.

**Comment by anasena** [comment]
Thank you for the help!
Looks like the ne120pg2_r05_oECv3 grid works on the F2010-CICE compset, so @jonbob is probably right. 
However, the ne120_r0125_oRRS18to6v3 still ends on a SIGTERM error even in this compset.
I was using the F20TR-CICE compset because I wanted to use the ghg/aerosol for 2000 with an SST forcing. 
Are there any simpler compset that you would recommend for the ne120 grid?
